DB/src/main/java/io/deephaven/stream/StreamToTableAdapter.java

```diff
@@ -21,6 +36,16 @@
 
     private final QueryTable table;
 
+    private final Index index;
```

Comment:
Can you humor me and rearrange these so that the final ones are before and separate from the mutable ones, and in some logical order?